webui/react/src/components/kit/Dropdown.tsx

```diff
@@ -1,7 +1,8 @@
 import { Popover as AntdPopover, Dropdown as AntDropdown } from 'antd';
 import { MenuProps as AntdMenuProps } from 'antd/es/menu/menu';
-import { PropsWithChildren, useMemo } from 'react';
+import { isValidElement, PropsWithChildren, useMemo } from 'react';
```

Comment:
> These APIs are exported from the react package, but they are not recommended for use in newly written code. See the linked individual API pages for the suggested alternatives.

https://react.dev/reference/react/legacy

prob we should avoid using `isValidElement`?

Reply:
i suppose it could just be `isForwardRef(children)` -- our UI kit `Button` won't be captured by `(isValidElement(children) && children?.type === 'button')` anyway since it's using `forwardRef`, I just kind of added that to cover all bases.